collectors/python.d.plugin/zscores/metadata.yaml

```
@@ -1,53 +1,161 @@
+
+        Every configuration JOB starts with a `job_name` value which will appear in the dashboard, unless a `name` parameter is specified.
       folding:
-        title: ''
+        title: "Config options"
         enabled: true
-      list: []
+      list:
+        - name: charts_regex
+          description: what charts to pull data for - A regex like `system\..*|` or `system\..*|apps.cpu|apps.mem` etc.
+          default_value: "system\\..*"
+          required: true
+        - name: train_secs
+          description: length of time (in seconds) to base calculations off for mean and stddev.
+          default_value: 14400
+          required: true
+        - name: offset_secs
+          description: offset (in seconds) preceding latest data to ignore when calculating mean and stddev.
+          default_value: 300
+          required: true
+        - name: train_every_n
+          description: recalculate the mean and stddev every n steps of the collector.
+          default_value: 900
+          required: true
+        - name: z_smooth_n
+          description: smooth the z score (to reduce sensitivity to spikes) by averaging it over last n values.
+          default_value: 15
+          required: true
+        - name: z_clip
+          description: cap absolute value of zscore (before smoothing) for better stability.
+          default_value: 10
+          required: true
+        - name: z_abs
+          description: "set z_abs: 'true' to make all zscores be absolute values only."
+          default_value: "true"
+          required: true
+        - name: burn_in
+          description: burn in period in which to initially calculate mean and stddev on every step.
+          default_value: 2
+          required: true
+        - name: mode
+          description: mode can be to get a zscore 'per_dim' or 'per_chart'.
+          default_value: per_chart
+          required: true
+        - name: per_chart_agg
+          description: per_chart_agg is how you aggregate from dimension to chart when mode='per_chart'.
+          default_value: mean
+          required: true
+        - name: update_every
+          description: Sets the default data collection frequency.
+          default_value: 5
+          required: false
+        - name: priority
+          description: Controls the order of charts at the netdata dashboard.
+          default_value: 60000
+          required: false
+        - name: autodetection_retry
+          description: Sets the job re-check interval in seconds.
+          default_value: 0
+          required: false
+        - name: penalty
+          description: Indicates whether to apply penalty to update_every in case of failures.
+          default_value: yes
+          required: false
     examples:
       folding:
         enabled: true
-        title: ''
-      list: []
+        title: "Config"
+      list:
+        - name: Default
+          description: Default configuration.
```

To have the example expanded add:
```suggestion
          description: Default configuration.
          folding:
            enabled: false
```